Enable setting max payload size for function (#8989)
kyma-project/kyma: docs/serverless/05-01-serverless-envs.md

---

> title: Environmental variables

Reviewer: ```suggestion
title: Environment variables
```

type: Configuration
---

> To configure the Serverless Function, override the default values of enviromental variables:

Reviewer: ```suggestion
To configure the Serverless Function, override the default values of these environment variables:
```

> | Environmental variable | Description                                                                   | Unit   | Default value |

Reviewer: ```suggestion
| Environment variable | Description                                                                   | Unit   | Default value |
```

| ---------------------- | ----------------------------------------------------------------------------- | ------ | ------------- |

> | **FUNC_TIMEOUT**       | Specifies the number of seconds to execute code before terminating execution. | Number | 180           |

Reviewer: ```suggestion
| **FUNC_TIMEOUT**       | Specifies the number of seconds in which a runtime must execute the code. | Number | `180`           |
```

> | **REQ_MB_LIMIT**       | Payload body size limit in megabytes                                          | Number | 1             |

Reviewer: ```suggestion
| **REQ_MB_LIMIT**       | Payload body size limit in megabytes                                          | Number |  `1`            |
```

> | **REQ_MB_LIMIT**       | Payload body size limit in megabytes                                          | Number |  `1`            |

Reviewer: ```suggestion
| **REQ_MB_LIMIT**       | Specifies the payload body size limit in megabytes.                                          | Number |  `1`            |
```

> See [this file](https://github.com/kubeless/runtimes/blob/master/stable/nodejs/kubeless.js) to get deeper understanding of how exactly those values are used. This Express server is internally used to run Functions.

Reviewer: ```suggestion
See [`kubeless.jsc`](https://github.com/kubeless/runtimes/blob/master/stable/nodejs/kubeless.js) to get a deeper understanding of how the Express server, that acts as a runtime, uses these values internally to run Functions.
```

> See the example of a function with such environmental variables set:

Reviewer: ```suggestion
See the example of a Function with these environment variables set:
```
